src/main/service/filterService.ts

```diff
@@ -1,11 +1,13 @@
 import {Court} from '../models/court';
 
+const filterNames = ['Jurisdiction', 'Region'];
+
 export class FilterService {
-  public getFilterValueOptions(filterName: string, list: Array<Court>): string[] {
+  private getFilterValueOptions(filterName: string, list: Array<Court>): string[] {
     return [...new Set(list.map(court => court[filterName.toLowerCase()]))];
   }
 
-  public buildFilterValueOptions(filterNames: string[], list, selectedFilters: string[]): object {
+  public buildFilterValueOptions(list, selectedFilters: string[]): object {
```

Comment:
based on your comment 'if we can start being little more accurate with return types'
shouldn't we do the same for attributes passed in?